_posts/2020-10-20-3.8.0-release-announcement.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Pulp 3.8.0 is Generally Available!
+author: Melanie Corr
+tags:
+  - announcement
+---
+
+The Pulp team are pleased to announce the release of Pulp 3.8.0!
+
+## Updates to Django settings
+
+* The Django `admin` site URL is now configurable using the new `ADMIN_SITE_URL` settings parameter [7637](https://pulp.plan.io/issues/7637).
+
+* As a follow up to [#7179](https://pulp.plan.io/issues/7179), which provided a default `DJANGO_SETTINGS_MODULE` variable, in this release, the default `DJANGO_SETTINGS_MODULE` variable is now set everywhere. This simplifies configuration because the **systemd** service no longer has to set this value [#7720](https://pulp.plan.io/issues/7720).
+
+## Allowed Content Checksums Validation
+
+In the previous release `ALLOWED_CONTENT_CHECKSUMS` variable was introduced so that users can specify and limit unwanted checksums in their environment.
+
+In this release, validation has been added so that if a user has Artifacts in their database and adds checksums to `ALLOWED_CONTENT_CHECKSUMS` that do not match the existing Artifacts, the error `ImproperlyConfigured` is thrown [#7487](https://pulp.plan.io/issues/7487).
+
+## Dynaconf Version Requirement
+
+Earlier versions of dynaconf proved buggy and error prone. In this release, the required dynaconf version is 3.1.2. [#7682](https://pulp.plan.io/issues/7682).
+
+## Bug Fixes
+
+* When a task was cancelled, it was logging as a failed task [#4559](https://pulp.plan.io/issues/4559)
+* When a task was cancelled, the status of all incomplete "progress reports" was remaining in an in-progress state [#4921](https://pulp.plan.io/issues/4921)
+* A fix has been added to handle duplicate content during synchronization and migration from Pulp 2 to Pulp 3 [#7147](https://pulp.plan.io/issues/7147)
+* Content streaming has been enabled for the `RepositoryVersionDistribution` parameter [#7568](https://pulp.plan.io/issues/7568)
```

Comment:
I struggle with this one, let's maybe ping @dkliban or @bmbouter . I think the changelog provided with the PR is not very user friendly. RepositoryVersionDistribution seems like an internal detail to me.

@dkliban, @bmbouter, is RepositoryVersionDistribution used when publication is not needed in a plugin and the latest repo version is distributed automatically?
How about "Content streaming has been enabled for the cases when auto-distribution feature is used"?

Reply:
Content streaming has been enabled for Distributions that use Repository Versions instead of Publications.

Maybe something like that?

Reply:
I like that. Users deal with all those entities and it's less ambiguous than "auto-distribution".

Reply:
Thank you!